server/handles/fsup.go

```diff
@@ -118,12 +140,17 @@ func FsForm(c *gin.Context) {
 		common.ErrorResp(c, err, 500)
 		return
 	}
+	if osFile != nil {
+		ss.SetTmpFile(osFile)
+	}
+
 	if asTask {
 		err = fs.PutAsTask(dir, ss)
 	} else {
+		defer ss.Close()
```

Comment:
It seems to be closed repeatedly here, as the Close has been called in op.Put

Reply:
it seems so, i'll remove this line since it's needed